### PR TITLE
cmd: add '--show-md' flag for 'dvc metrics show' command (#4770)

### PR DESCRIPTION
**Pull request by DiPaolo**
Matches iterative/dvc/pull/5161

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
Hi @DiPaolo can you please link to the core repo PR that implements this? Thanks

**Comment by jorgeorpinel** [comment]
Nevermind, found it: iterative/dvc/pull/5161

File: content/docs/command-reference/metrics/show.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Print [metrics](/doc/command-reference/metrics), with optional formatting.
 
 ```usage
 usage: dvc metrics show [-h] [-q | -v] [-a] [-T] [--all-commits] [-R]
-                        [--show-json]
+                        [--show-json][--show-md]
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/show.md:9]:
```suggestion
                        [--show-json] [--show-md]
```

**Review by jorgeorpinel — APPROVED** [review]
Thanks @DiPaolo . Please try to let us know when the core PR is merged 🙂

**Comment by efiop** [comment]
@jorgeorpinel Merged. I see that linter checks fail though. @DiPaolo Mind fixing them, please?

**Comment by DiPaolo** [comment]
@jorgeorpinel iterative/dvc/pull/5161 merged

**Comment by shcheklein** [comment]
@DiPaolo merged the restyled version! thanks! 🙏

**Comment by jorgeorpinel** [comment]
Thanks guys!

**Pull request closed without merging**